Extensions/LinkedObjects/tests/linkedobjects.spec.js

```diff
@@ -1,106 +1,302 @@
+    const object2B = new gdjs.RuntimeObject(runtimeScene, {
+      name: object2Names[1],
+      type: '',
+      behaviors: [],
+      effects: [],
+    });
+    // @ts-ignore - missing properties.
+    const object2C = new gdjs.RuntimeObject(runtimeScene, {
+      name: object2Names[2],
+      type: '',
+      behaviors: [],
+      effects: [],
+    });
 
-    var linkedObjects = manager.getObjectsLinkedWith(object1A);
-    expect(linkedObjects.length).to.be(1);
-    expect(linkedObjects[0]).to.be(object2A);
+    // @ts-ignore - missing properties.
+    const object3A = new gdjs.RuntimeObject(runtimeScene, {
+      name: object3Names[0],
+      type: '',
+      behaviors: [],
+      effects: [],
+    });
 
-    linkedObjects = manager.getObjectsLinkedWith(object2A);
-    expect(linkedObjects.length).to.be(1);
-    expect(linkedObjects[0]).to.be(object1A);
-  });
-  it('can link more objects', function () {
-    manager.linkObjects(object1A, object2A); //Including the same objects as before
-    manager.linkObjects(object1A, object2B);
-    manager.linkObjects(object1A, object2C);
+    // @ts-ignore - missing properties.
+    const object3B = new gdjs.RuntimeObject(runtimeScene, {
+      name: object3Names[1],
+      type: '',
+      behaviors: [],
+      effects: [],
+    });
+    // @ts-ignore - missing properties.
+    const object3C = new gdjs.RuntimeObject(runtimeScene, {
+      name: object3Names[2],
+      type: '',
+      behaviors: [],
+      effects: [],
+    });
 
-    var linkedObjects = manager.getObjectsLinkedWith(object1A);
-    expect(linkedObjects.length).to.be(3);
+    runtimeScene.addObject(object1A);
+    runtimeScene.addObject(object1B);
+    runtimeScene.addObject(object1C);
 
-    linkedObjects = manager.getObjectsLinkedWith(object2C);
-    expect(linkedObjects.length).to.be(1);
-    expect(linkedObjects[0]).to.be(object1A);
-  });
-  it('supports removing links', function () {
-    manager.removeLinkBetween(object1A, object2B);
-    var linkedObjects = manager.getObjectsLinkedWith(object1A);
-    expect(linkedObjects.length).to.be(2);
+    runtimeScene.addObject(object2A);
+    runtimeScene.addObject(object2B);
+    runtimeScene.addObject(object2C);
 
-    manager.linkObjects(object2B, object2C);
-    manager.removeAllLinksOf(object1A);
-    manager.removeAllLinksOf(object1A);
+    runtimeScene.addObject(object3A);
+    runtimeScene.addObject(object3B);
+    runtimeScene.addObject(object3C);
 
-    linkedObjects = manager.getObjectsLinkedWith(object1A);
-    expect(linkedObjects.length).to.be(0);
+    /**
+     * @param {gdjs.RuntimeObject} object
+     * @param {{objectName: string, picked: gdjs.RuntimeObject[]}[]} pickedObjects
+     * @returns a selection of picked objects
+     */
+    const pickObjectsLinkedTo = (object, pickedObjects) => {
+      const objectsLists = new Hashtable();
+      for (const pickedObject of pickedObjects) {
+        objectsLists.put(pickedObject.objectName, pickedObject.picked);
+      }
```

Comment:
Isn't using `Hashtable.newFrom({ 'obj1': [....] })` as easy as using a custom structure "pickedObjects"? 
It's creating another abstraction which was making me wonder what this was providing.